### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: translations/nl.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl">
+<context>
+    <name>PagePower</name>
+    <message>
+        <source>Suspend</source>
+        <translation>Pauzestand</translation>
```
Comment (QazCetelic) [inline translations/nl.ts:8]: I think "sluimerstand" could be more fitting.

Comment (Vistaus) [reply translations/nl.ts:8]: I've never heard that term before. “Pauzestand” is the regular term, at least on Linux. So I'd like to keep it consistent.

**Comment by EricKotato** [comment]
Contributing .qm file wasn't neccessary, but thanks.

**Comment by Vistaus** [comment]
> Contributing .qm file wasn't neccessary, but thanks.

Oh, I saw the ru.qm file, so I thought it was necessary, but alright. 🙂

**Comment by EricKotato** [comment]
I usually generate them from .ts files myself. Though to be honest, it's better to make a GitHub Action, so it will be generated automatically, but that's the story for another day.

**Pull request merged**